packages/owl-bot/test/owl-bot.ts

```diff
@@ -1601,23 +1633,23 @@ describe('owlBot', () => {
         number: 33,
         head: {
           repo: {
-            full_name: 'bcoe/owl-bot-testing',
+            full_name: 'googleapis/owl-bot-testing',
           },
           ref: 'abc123',
         },
         base: {
           ref: 'main',
           repo: {
-            full_name: 'bcoe/owl-bot-testing',
+            full_name: 'googleapis/owl-bot-testing',
           },
         },
       },
     };
     const githubMock = nock('https://api.github.com')
-      .get('/repos/bcoe/owl-bot-testing/pulls/33')
+      .get('/repos/googleapis/owl-bot-testing/pulls/33')
       .reply(200, payload.pull_request)
       .get(
-        '/repos/bcoe/owl-bot-testing/contents/.github%2F.OwlBot.lock.yaml?ref=abc123'
+        '/repos/googleapis/owl-bot-testing/contents/.github%2F.OwlBot.lock.yaml?ref=abc123'
```

Comment:
maybe add one test with `googlecloudplatform`, just to test both cases.

Reply:
Added one test with `GoogleCloudPlatform` (CamelCased)